### PR TITLE
Update versioning-sdk-msbuild-vs.md

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
Updating this page per changes made by Marc to the version of this doc in the first party wiki...
https://devdiv.visualstudio.com/FirstPartyEngagements/_wiki/wikis/FirstPartyEngagements.wiki?wikiVersion=GBwikiMaster&pagePath=/NET%20Core%20Adoption%20Guide/NET%20Core%20Adoption%20Resources/Guides/MSBuild%2C%20VS%2C%20.NET%20SDK%20Versioning&_a=compare&version=4e1d045532d3fa77cb1a6ffb39b533bb158e89b4

### CODE REVIEW

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 1]
```diff
@@ -25,13 +25,14 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
 | 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
-| 5.0.1xx          | 16.8               | November '20 | Mar '21   |
+| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
 | 5.0.2xx          | 16.9               | March '21    | Aug '22   |
 | 5.0.3xx          | 16.10              | May '21      | Aug '21   |
 | 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |
-| 6.0.1xx          | 17.0**             | Nov. '21     | Jul '23   |
-| 6.0.2xx          | 17.1               |  '21         |           |
-| 6.0.3xx          | 17.2               |  '21         |           |
+| 6.0.100-rc1      | 17.0-preview 4     | Sep '21      | N/A       |
```
Comment (marcpopMSFT) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:32]: please remove 6.0.100-rc1

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 2]
```diff
@@ -25,13 +25,14 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
+| 6.0.100          | 17.0**             | Nov '21      | May '23   |
```
Comment (marcpopMSFT) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:33]: For these three, please remove the May dates. Those are all dependent on when 17.2 ships and we have not committed to a specific date yet (I wouldn't want to publish something and then have it move).

File: docs/core/porting/versioning-sdk-msbuild-vs.md (hunk 1 above)
Comment (StephenBonikowsky) [reply docs/core/porting/versioning-sdk-msbuild-vs.md]: done

File: docs/core/porting/versioning-sdk-msbuild-vs.md (hunk 2 above)
Comment (StephenBonikowsky) [reply docs/core/porting/versioning-sdk-msbuild-vs.md]: done, I replaced them with tbd.

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 3]
```diff
@@ -25,13 +25,13 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
 | 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
-| 5.0.1xx          | 16.8               | November '20 | Mar '21   |
+| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
 | 5.0.2xx          | 16.9               | March '21    | Aug '22   |
 | 5.0.3xx          | 16.10              | May '21      | Aug '21   |
 | 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |
-| 6.0.1xx          | 17.0**             | Nov. '21     | Jul '23   |
-| 6.0.2xx          | 17.1               |  '21         |           |
-| 6.0.3xx          | 17.2               |  '21         |           |
+| 6.0.100          | 17.0**             | Nov '21      | tbd       |
```
Comment (gewarren) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:32]:
```suggestion
| 6.0.100          | 17.0**             | Nov '21      | TBD       |
```

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 4]
```diff
@@ -25,13 +25,13 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
+| 6.0.200          | 17.1               | Feb '22      | tbd       |
```
Comment (gewarren) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:33]:
```suggestion
| 6.0.200          | 17.1               | Feb '22      | TBD       |
```

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 5]
```diff
@@ -25,13 +25,13 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
+| 6.0.300          | 17.2^              | tbd          | tbd       |
```
Comment (gewarren) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:34]:
```suggestion
| 6.0.300          | 17.2^              | TBD          | TBD       |
```

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 6]
```diff
@@ -40,6 +40,8 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
 >
 > \*\* When .NET 6 releases, the goal is for the .NET SDK to be functional in version 16.11 for **downlevel** targeting. This means that you're not forced to update your SDK and Visual Studio versions simultaneously. However, you won't be able to target .NET 6 because of limitations in 6.0 features and C#10 features in version 16.11. This compatibility is specifically for targeting 5.0 and below.
```
Comment (gewarren) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:42]:
```suggestion
>
```

File: docs/core/porting/versioning-sdk-msbuild-vs.md [hunk 7]
```diff
@@ -40,6 +40,8 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
...
 
+^6.0.300 and newer SDKs will require a minimum VS version of 17.0
```
Comment (gewarren) [inline docs/core/porting/versioning-sdk-msbuild-vs.md:43]:
```suggestion
> ^ 6.0.300 and newer SDKs require a minimum Visual Studio version of 17.0.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**